pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -267,19 +264,25 @@ func TestVolumeBinding(t *testing.T) {
 			for _, pv := range item.pvs {
 				client.CoreV1().PersistentVolumes().Create(ctx, pv, metav1.CreateOptions{})
 			}
+
+			t.Log("Start informer factory after initialization")
+			informerFactory.Start(ctx.Done())
```

Comment:
see explanation here: https://github.com/kubernetes/kubernetes/issues/93009#issuecomment-667527689

In my understanding, with a real client and the real API Server events should not be lost.

Reply:
> However, with the fake client, events may be lost in testing.

If you need to act upon the API change events on a UT, I think you have to try prepend/add reactor for a fake client. You can search:

- client.PrependReactor()
- client.PrependWatchReactor()

However, if the test just relies on the a fixed state of all initialized API objects (i think it's the case, correct me if I'm wrong), the current change (start informerFactory and wait for its cache to be synced) should be good enough.

Reply:
> However, if the test just relies on the a fixed state of all initialized API objects
yes, it is